tools/pagerduty-migrator/migrator/resources/integrations.py

```diff
@@ -52,7 +53,14 @@ def migrate_integration(integration: dict, escalation_policies: list[dict]) -> N
 def create_integration(
     name: str, integration_type: str, escalation_chain_id: str
 ) -> None:
-    payload = {"name": name, "type": integration_type, "team_id": None}
+
+    DEFAULT_INTEGRATION_TYPE = ["alertmanager", "grafana", "webhook", "kapacitor", "elastalert", "zabbix"]
+    fallback_to_integration_type = os.environ.get('FALLBACK_TO_WEBHOOK_INTEGRATION_TYPE')
```

Comment:
I think this should be in [config.py](https://github.com/grafana/oncall/blob/dev/tools/pagerduty-migrator/migrator/config.py), so it's consistent with other constants.